python/trigger/test_channel_filter/toplevel.py

```diff
@@ -0,0 +1,56 @@
+import json
+import os
+import rich.traceback
+from rich.console import Console
+
+from appfwk.system import System
+
+# Add -h as default help option
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+console = Console()
+
+import click
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('-s', '--slowdown-factor', default=1.0)
+@click.option('-f', '--input-file', type=click.Path(exists=True, dir_okay=False), multiple=True)
+@click.option('--keep-collection', is_flag=True, default=True, show_default=True, help="Keep collection TPs")
+@click.option('--keep-induction', is_flag=True, default=True, show_default=True, help="Keep induction TPs")
```

Comment:
```py
@click.option('--keep-collection/--discard-collection', default=True, show_default=True, help="Keep collection TPs")
@click.option('--keep-induction/--discard-induction', default=True, show_default=True, help="Keep induction TPs")
```

Reply:
Nice! I didn't know about this feature. I've added this.